mkinitramfs-ll.sh

```diff
@@ -25,6 +25,7 @@ usage() {
   -c, --compressor='gzip -9'  Use 'gzip -9' compressor instead of default
```

Comment:
rm /boot/initramfs*
mkinitramfs-ll.sh --hook=zfs --luks --gpg --compressor=none
ls /boot ex: if only initramfs-4.7.10-hardened.cpio

i generate a grub.cfg with mkconfig -o /boot/grub/grub.cfg
and grub find only the vmlinuz, no initrd.